src/components/kid/create/content/Step2.tsx

```diff
@@ -0,0 +1,58 @@
+import { useNavigate } from 'react-router-dom';
+import { useAppDispatch } from '../../../../store/app/hooks';
+import styled from 'styled-components';
+import { dispatchItemName } from '../../../../store/slices/challengePayloadSlice';
+import SelectItemNameButton from '../SelectItemNameButton';
+import a1 from '../../../../assets/icons/itemName/a1.svg';
+import a2 from '../../../../assets/icons/itemName/a2.svg';
+import a3 from '../../../../assets/icons/itemName/a3.svg';
+import b1 from '../../../../assets/icons/itemName/b1.svg';
+import b2 from '../../../../assets/icons/itemName/b2.svg';
+import b3 from '../../../../assets/icons/itemName/b3.svg';
+import c1 from '../../../../assets/icons/itemName/c1.svg';
+import c2 from '../../../../assets/icons/itemName/c2.svg';
+import c3 from '../../../../assets/icons/itemName/c3.svg';
+
+const itemNameList = [
```

Comment:
```suggestion
const items = [
```
참고: https://www.notion.so/pocketmoney/Coding-Convention-Git-Flow-150511390b964d7db15c763c41c8f35a#330c55912e044054bfed7d70d59bcbf5

Reply:
백엔드 명세에 계약 상품으로 써있는 이름이 itemName이어서, itemNames로 수정하겠슴다